core/device_plugin_container.cpp.in

```diff
@@ -43,5 +48,19 @@ void DevicePluginContainer::deinit_plugins()
     }
 }
 
+void DevicePluginContainer::enable_plugins()
+{
+    for (auto it = _plugin_impl_list.begin(); it != _plugin_impl_list.end(); ++it) {
```

Comment:
What about a range-based for here? I find it easier to read.

```
for (auto plugin_impl : _plugin_impl_list) {
    plugin_impl->enable();
}
```

Same comment for the other `for` loops in this file, actually. If it is compiling on all platforms, I would be in favor of the range-based for loop.

Reply:
Definitely, thanks!